sympy/core/basic.py

```diff
@@ -1921,7 +1932,7 @@ def _atomic(e, recursive=False):
     seen = set()
     if isinstance(e, Basic):
         free = getattr(e, "free_symbols", None)
-        if free is None:
```

Comment:
None was the default for an object that didn't have the `free_symbols` attribute. You are now returning `{e}` if `e` is a SymPy object with no free symbols.

Reply:
I'm actually not sure where all these changes came from. I had these commits sitting around and just pushed them to see what would happen with the tests.